core/src/main/resources/error/error-classes.json

```diff
@@ -1207,6 +1207,12 @@
     ],
     "sqlState" : "42K03"
   },
+  "REPEATED_CLAUSE" : {
+    "message" : [
+      "The <clause> clause may be used at most once per <operation> operation."
+    ],
+    "sqlState" : "2202G"
```

Comment:
2202G is: invalid repeat argument in a sample clause is quite specific.
```suggestion
    "sqlState" : "42616"
```

Reply:
Just applied the suggestion!

Btw, seems like 42616 is missing in https://github.com/apache/spark/tree/master/core/src/main/resources/error.
Is it intentionally excluded or do we want to add it?

Reply:
I mistypes. Should have been 42614 Duplicate clause.
42616 is Db2 LUW specific (I drew on DB2 zOS to fill the table

Reply:
Thanks for the correction! Just updated to 42614.